code/modules/mob/living/simple_animal/hostile/mothership_creatures.dm

```diff
@@ -447,7 +451,7 @@
 
 /mob/living/simple_animal/hostile/retaliate/gym_rat/attackby(var/obj/item/O as obj, var/mob/user as mob) // Feed the gym rat some food
 	if(stat == CONSCIOUS)
-		if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/cheesewedge)) // Cheese heals it a bit
+		if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/cheesewedge)) // Cheesewedges heal it a bit
```

Comment:
Should probably include full cheese wheels, not like it can't eat them.

Reply:
Should be fixed.